### ISSUE TITLE
List errors question

### ISSUE THREAD

**Issue by xHN35RQ** [wooorm/mdast-lint#28]
In Atom with `linter-markdown` given the following list with sub-items which I assume is valid GFM markdown. (derived from https://github.com/wooorm/mdast/issues/29#issuecomment-114027854)

```
-   aa
    -   ab
        -   ac
-   ba
    -   bb
        -   bc
-   ca
    -   cb
        -   cc
```

I see the following errors:

```
Error
Missing blank line before block node at line 2 col 5
Error
Missing blank line before block node at line 3 col 9
Error
List item should be loose, isn’t at line 3 col 15
Error
Missing blank line before block node at line 5 col 5
Error
Missing blank line before block node at line 6 col 9
Error
List item should be loose, isn’t at line 6 col 15
Error
Missing blank line before block node at line 8 col 5
Error
Missing blank line before block node at line 9 col 9
```

I'm curious as to why I'm receiving these errors. Is this a bug, or am I misunderstanding something here?

Thanks!

**Comment by wooorm** [comment]
Thanks for the question!

Not to worry, your markdown is indeed valid. However, mdast-lint (just like eslint and often other linters), highlight potential (style-guide) violations. In this case, two rules are triggered [no-missing-blank-lines](https://github.com/wooorm/mdast-lint/blob/master/doc/rules.md#no-missing-blank-lines) and [list-item-spacing](https://github.com/wooorm/mdast-lint/blob/master/doc/rules.md#list-item-spacing), which both originate in from http://www.cirosantilli.com/markdown-style-guide/.

Read them as “warnings”, not “errors” :wink:

**Comment by xHN35RQ** [comment]
Ah ok, I wondered if that might be the case. Thanks so much for your reply.

For now I'm surpressing these warnings by placing the following into an `.mdastrc` file in my current project directory where I'm working with several markdown files:

```
{
  "plugins": {
    "lint": {
        "no-missing-blank-lines": false,
        "list-item-spacing": false
    }
  }
}
```

Thanks again for your help.

**Comment by wooorm** [comment]
mdast-lint is quite opinionated on some points, in others it detects the style which you’re using and just detects violations.

No problem! Good luck!
